jax/experimental/jax2tf/examples/tf_js/quickdraw/README.md

```diff
@@ -0,0 +1,38 @@
+# Quickdraw
+
+This directory contains the code necessary to train a CNN on the Quickdraw
+dataset using Flax, and subsequently convert it using jax2tf into a model that
+can be loaded in TF.js.
+
+## Training the model
+
+To train and export the model yourself, simply run
+```bash
+$ python3 quickdraw.py
+```
+
+The dataset will be downloaded directly into a `data/` directory in the
+`/tmp/jax2tf/tf_js_quickdraw` directory; by default, the model is configured to
+classify inputs into 100 different classes, which corresponds to a dataset of
+roughly 11 Gb. This can be tweaked by modifying the value of the `NB_CLASSES`
+global variable in `quickdraw.py` (max number of classes: 100). Only the files
+corresponding to the first `NB_CLASSES` classes in the dataset will be
+downloaded.
+
+The training loop runs for 5 epochs, and the model as well as its equivalent
+TF.js-loadable model are subsequently saved into `/tmp/jax2tf/tf_js_quickdraw`.
+
+## Interacting with the model
+
+Here is a (third party) [demo](https://zaidalyafeai.github.io/sketcher/) of how
+interacting with the trained model looks like.
+
+If you want to try it yourself, the trained model along with third party front
```

Comment:
You said that there is a readme in the tarball. Perhaps it is worth saying here also that the tarball has the same exact files as the above linked website, with one line change to ..., along with the trained weights. Do you happen to have training accuracy numbers?

Reply:
I will upgrade. Yes, I have training accuracy numbers:
```
Training set accuracy 0.7825493812561035
Test set accuracy 0.764135718345642
```
where "accurate" means "the prediction with the highest likelihood is the expected class".

Reply:
Please update the accuracy numbers in the README